### PR TITLE
new lesson abstraction

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,39 @@
...
 
-## Conclusion
+- An abstract class may define instance and static variables that are inherited by subclasses.
+- An interface can define constants variables (static and final), but not instance or non-final static variables.
+- Neither an interface nor an abstract class can be instantiated using the `new` operator.
+- An abstract class can have parameterized constructors and the default constructor is always present in an abstract class.
+  - Abstract class constructors are called by subclass constructors to initialize inherited state.
+- An interface can't have a constructor.
+- An abstract class may define any number of concrete or abstract methods, including instance and static methods.
+- An interface may define any number of abstract, default, or static methods.
```
Comment (kcleland0818) [inline README.md]: Suggestion: Maybe a table showing the differences might be easier to visualize?

**Review by kcleland0818 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: good idea, updated lesson.

**Review by kcleland0818 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,31 +2,38 @@
...
-appropriate to help with the reader's orientation with the material and future
-referencing.
+## Comparing Abstract Classes and Interfaces
 
-## Conclusion
+| Abstract Class                                                                                | Interface                                                                                               |
+|-----------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|
+| May define instance and static variables                                                      | May define constant variables (static+final). Instance and non-final static variables are not allowed.  |
+| Can't instantiate using `new` operator.                                                       | Can't instantiate using `new` operator.                                                                 |
+| Can define parameterized constructors. Always has a default constructor.                      | Can't define a constructors.  Does not have a default constructor.                                      |
```
Comment (developerjaym) [inline README.md]: Consider removing 'a' from 'a constructors'

Comment (linda-seiter) [reply README.md]: fixed

**Review by kcleland0818 — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**